macros/system/configuration/upload_edr_configuration.sql

```diff
@@ -0,0 +1,133 @@
+{% macro upload_tables_configuration() %}
+    {% if execute %}
+        {% set nodes = elementary_data_reliability.get_nodes_from_graph() %}
+        {% set table_monitors = [] %}
+        {% for node in nodes | selectattr('resource_type', 'in', 'seed,source,model') -%}
+            {% set table_config_dict = elementary_data_reliability.get_table_config(node) %}
+            {% if table_config_dict is not none %}
+                {% do table_monitors.append(table_config_dict) %}
+            {% endif %}
+        {% endfor %}
+        {% if table_monitors|length > 0 %}
```

Comment:
You can just do 
{% if table_monitors|length %}
Do you prefer to be explicit?

Reply:
Yeah, I prefer to be explicit